packages/preset-create-react-app/src/index.ts

```diff
@@ -109,12 +109,30 @@ export const webpack = (
     },
   );
 
+  // A temporary fix to align with Storybook 6.
+  const { typescriptOptions } = options;
+  const tsDocgenRule =
+    typescriptOptions &&
+    typescriptOptions.reactDocgen === 'react-docgen-typescript'
```

Comment:
```suggestion
    (typescriptOptions.reactDocgen === 'react-docgen-typescript' || typeof typescriptOptions.reactDocgen === 'undefined')
```

Reply:
With v6 this is always defined, because of our defaults.

Reply:
Aha right